FND/Accounting/ISO4217-CurrencyCodes.rdf

```diff
@@ -34,12 +34,15 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/">
 		<rdfs:label>ISO 4217-1 Currency Codes Ontology</rdfs:label>
 		<dct:abstract>This ontology represents the subset of the ISO 4217 standard that include the actual currency codes.</dct:abstract>
+		<dct:issued rdf:datatype="&xsd;dateTime">2021-10-01T00:00:00&gt;</dct:issued>
 		<dct:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dct:license>
-		<sm:contentLanguage rdf:datatype="&xsd;anyURI">http://www.w3.org/standards/techs/owl#w3c_all</sm:contentLanguage>
+		<sm:contentLanguage rdf:datatype="&xsd;anyURI">https://www.w3.org/TR/owl2-quick-reference/</sm:contentLanguage>
 		<sm:contributor>Adaptive, Inc.</sm:contributor>
 		<sm:contributor>Thematix Partners LLC</sm:contributor>
-		<sm:copyright>Copyright (c) 2015-2020 EDM Council, Inc.</sm:copyright>
-		<sm:copyright>Copyright (c) 2015-2020 Object Management Group, Inc.</sm:copyright>
+		<sm:contributor>agnos.ai UK Ltd.</sm:contributor>
+		<sm:copyright>Copyright (c) 2015-2022 EDM Council, Inc.</sm:copyright>
+		<sm:copyright>Copyright (c) 2015-2022 Object Management Group, Inc.</sm:copyright>
+		<sm:copyright>Copyright (c) 2022 agnos.ai UK Ltd.</sm:copyright>
```

Comment:
Up to this point we followed strictly https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#ontology-header-information and copyrighted FIBO ontologies only to EDMC and, in some cases to OMG.

I am happy to recognize agnos.ai contribution as we did in the past, i.e., using sm:contributor, but copyrighting to them requires broader discussion.

@ElisaKendall could you comment?

Reply:
@rivettp Actually, if you are going to do that, then many of the ontologies that I personally contributed, including this one that I constructed manually before you generated it, should also include a Thematix copyright since that work was done when I was not part of the EDMC team officially.  This is why we added ourselves as contributors to some of these ontologies.  If you intend to copyright all of the ones you've generated, then you'll need to add Thematix since I created them manually before you were able to generate them.  I'm not advocating for that, but we should apply the rules consistently.  You could, alternatively, add yourself as agnos.ai in addition to Adaptive as a contributor.

Reply:
Added Thematix copyright. 
The broader question of attributing copyright for other ontologies is a broader question outside the scope of this PR.